lib/circuit.js

```diff
@@ -316,15 +327,15 @@ class CircuitBreaker extends EventEmitter {
   }
 }
 
-function handleError (error, circuit, timeout, args, resolve, reject) {
+function handleError (error, circuit, timeout, args, latencyEndTime, resolve, reject) {
   clearTimeout(timeout);
-  fail(circuit, error, args);
-  const fb = fallback(circuit, error, args);
+  fail(circuit, error, args, latencyEndTime);
+  const fb = fallback(circuit, error, args, latencyEndTime);
   if (fb) resolve(fb);
   else reject(error);
 }
 
-function fallback (circuit, err, args) {
+function fallback (circuit, err, args, latencyEndTime) {
```

Comment:
ah,  doesn't look like it is,  i think i put it in there because i had a thought about storing the latency during the fallback event, but i don't think we need to do that,  i'll remove it for now